examples/hello-world/README.md

```diff
@@ -137,7 +137,7 @@ Next, you can check the list of running Apps:
 
 ```shell
 $ docker app ls
-INSTALLATION   APPLICATION         LAST ACTION   RESULT    CREATED              MODIFIED             REFERENCE
+RUNNING APP    APP NAME            LAST ACTION   RESULT    CREATED              MODIFIED             REFERENCE
```

Comment:
Wondering if `APP NAME` should become `APP IMAGE` at some point

Reply:
Here this is actually the app name coming the metadata, so really this has nothing to do with the name you give to the app image when you build

Reply:
Still unclear to me what this metadata name is for. I can't imagine much scenario where it's anything else but the repository name image is published to.